src/analysis/special_functions/log/base.lean

```diff
@@ -52,6 +52,39 @@ by simp_rw [logb, log_div hx hy, sub_div]
 
 @[simp] lemma logb_inv (x : ℝ) : logb b (x⁻¹) = -logb b x := by simp [logb, neg_div]
 
+lemma inv_logb (a b : ℝ) : (logb a b)⁻¹ = logb b a := by simp_rw [logb, inv_div]
+
+theorem inv_logb_mul_base {a b : ℝ} (c : ℝ) (h₁ : a ≠ 0) (h₂ : b ≠ 0):
```

Comment:
A final pair of nits:
* Spaces before `:`s
* Put the arguments in the order they appear in the lemma to make it easier to guess

```suggestion
theorem inv_logb_mul_base {a b : ℝ} (ha : a ≠ 0) (hb : b ≠ 0) (c : ℝ) :
```

`c` appears after `a` and `b`, so it makes sense for it to be after the other arguments about `a` and `b`.

Same throughout.